jme3-core/src/main/java/com/jme3/texture/TextureArray.java

```diff
@@ -151,4 +156,41 @@ public void setWrap(WrapMode mode) {
         this.wrapS = mode;
         this.wrapT = mode;
     }
+
+    @Override
+    public void write(JmeExporter e) throws IOException {
+        super.write(e);
+        OutputCapsule capsule = e.getCapsule(this);
+        capsule.write(wrapS, "wrapS", WrapMode.EdgeClamp);
+        capsule.write(wrapT, "wrapT", WrapMode.EdgeClamp);
+    }
+
+    @Override
+    public void read(JmeImporter importer) throws IOException {
+        super.read(importer);
+        InputCapsule capsule = importer.getCapsule(this);
+        wrapS = capsule.readEnum("wrapS", WrapMode.class, WrapMode.EdgeClamp);
+        wrapT = capsule.readEnum("wrapT", WrapMode.class, WrapMode.EdgeClamp);
+    }
+
+    @Override
+    public boolean equals(Object other) {
```

Comment:
The equals() method just needs updated to include the == check first:

```
       if (this == obj) {
            return true; 
        }
```

Otherwise everything looks good with the PR and it should be ready to merge after this change.

Reply:
Good catch!
I'll try to fix it as soon as I can.
I might not find the time in the next few days so if you want to make the change quick please go ahead.
thx

Reply:
Sounds good, I've went ahead and made the change for you, so everything should be good and ready to merge now. Thanks for your contribution! 

And I also owe you a second thanks, since reporting this issue also helped me figure out how to properly save my project's Texture Arrays